### ISSUE TITLE
Update Jasmin example config in config-user.yml to include native6 project

### ISSUE THREAD

**Issue by zklaus** [ESMValGroup/ESMValCore#1246]
**Is your feature request related to a problem? Please describe.**
The Jasmin config does not yet include the `native6` project. This leads to some unexpected error messages.

**Comment by zklaus** [comment]
Or perhaps this is because the RAWOBS are not available on Jasmin? @valeriupredoi, what's the situation?

**Comment by valeriupredoi** [comment]
no RAWOBS on JASMIN indeed, @jvegasbsc had a shy tentative of starting one but I barked at him so badly (no disk space) that the poor fella deleted it. I don't think we can accommodate one, there is really not much room, currently `esmeval` is like this:
```bash
[valeriu@sci1 ~]$ df -h /gws/nopw/j04/esmeval
Filesystem                           Size  Used Avail Use% Mounted on
quobyte@sds.jc.rl.ac.uk/gws_esmeval  4.6T  1.3T  3.3T  28% /gws/nopw/j04/esmeval
```
and we should leave plenty of room for upcoming cmorized datasets; to get another group worspace for the same purpose will be virtually impossible. How much native6 data do you think we should host on JASMIN, if at all?

**Comment by zklaus** [comment]
So we use around one quarter of the diskspace and have 3.3 terabytes available still? I think that's a solid case for including at least the native6 datasets, i.e. those that we support natively. After all, this will be more and more of the most interesting ones. For example, I think people will start to get cross if we don't offer ERA5.

Perhaps we should have a separation of the RAWOBS in native6 and rest and at least mirror the native6 part?

**Comment by valeriupredoi** [comment]
am all for it, if you could give me an estimate of the native6 size, we want to get ERA5 (currently have a puny 40 years of `pr` :grin: ) and I suspect there's plenty more on DKRZ -> @remi-kazeroni would be able to tell you; so that'd be three big chunks: one for ERA5, one for upcoming OBS in cmorized format and one for native6, say 1T each?

**Comment by remi-kazeroni** [comment]
On Mistral we have 15T in the RAWOBS folder:
```bash
$ du -sh RAWOBS/Tier*
9.3T    RAWOBS/Tier2
4.8T    RAWOBS/Tier3
```
and regarding native6 datasets:
```bash
$ du -sh RAWOBS/Tier3/ERA*
2.9T    RAWOBS/Tier3/ERA5
533G    RAWOBS/Tier3/ERA-Interim
91G     RAWOBS/Tier3/ERA-Interim-Land
```

Mirroring all ERA5 data to Jasmin would require almost all free disk space remaining... Without getting more disk space, we would first need to select what is needed for Jasmin. For info:
```bash
$ du -sh RAWOBS/Tier3/ERA5/v1/*
2.0T    RAWOBS/Tier3/ERA5/v1/1hr
712G    RAWOBS/Tier3/ERA5/v1/1hrPt
17G     RAWOBS/Tier3/ERA5/v1/fx
202G    RAWOBS/Tier3/ERA5/v1/mon
```
and the [variables](https://docs.esmvaltool.org/en/latest/input.html#supported-datasets)

**Comment by jvegasbsc** [comment]
ERA5 is already at jasmin in `/badc/ecmwf-era5`. We do not need the `RAWOBS` for this

**Comment by valeriupredoi** [comment]
thanks @jvegasbsc - do we have this written anywhere in the documentation and/or the cmorizer docstring? Doing an e.g. `ncdump -h /badc/ecmwf-era5/data/enda/an_sfc/2020/01/31/ecmwf-era5_enda_an_sfc_202001311500.mem6.2t.nc` shows me the data is slightly not cmorized :grin:

**Comment by jvegasbsc** [comment]
Yes, that's why we should configure the `native6` to work with that. I thinl @bouweandela  told me that it should work that way

**Comment by bouweandela** [comment]
It should work if the ERA5 time frequency is hourly or monthly (what does the `mem6` mean in the DRS above?), unless you want to use another native6 dataset that is not ERA5 at the same time (see the now-closed #494).

To be really user-friendly, the era5 variable name (and possibly other facets in the DRS) will need to be automatically added to the recipe, see #1125 for a start on that. Until that is done, they need to be specified in the recipe.

**Comment by bouweandela** [comment]
> On Mistral we have 15T in the RAWOBS folder:
>
>$ du -sh RAWOBS/Tier*
>9.3T    RAWOBS/Tier2
>4.8T    RAWOBS/Tier3
>
>and regarding native6 datasets:
>
>$ du -sh RAWOBS/Tier3/ERA*
>2.9T    RAWOBS/Tier3/ERA5

Note that the ERA5 data on Mistral appears to be far from complete, there are several recipes failing to run because of missing ERA5 data: https://github.com/ESMValGroup/ESMValTool/issues/2198#issuecomment-872966389. @remi-kazeroni Any chance of adding the missing data?

**Comment by valeriupredoi** [comment]
> what does the mem6 mean in the DRS above?

no idea, I just grabbed a random file I could find there just to see what's the hap with the data, I am by far the least knowledgeable one amongst us when it comes to ERA5 :grin:

**Comment by bouweandela** [comment]
> Doing an e.g. ncdump -h /badc/ecmwf-era5/data/enda/an_sfc/2020/01/31/ecmwf-era5_enda_an_sfc_202001311500.mem6.2t.nc shows me the data is slightly not cmorized :grin:

This is of course the point of supporting the data in it's native format (see [fixes](https://github.com/ESMValGroup/ESMValCore/blob/main/esmvalcore/cmor/_fixes/native6/era5.py), it is only slightly non-cmor) and like this we avoid having to copy this massive dataset. This file contains `2t`, i.e. what we know as `tas`.

**Comment by jvegasbsc** [comment]
@lpcaron needs this for a paper's revision, so I am starting to work on it. I will keep you updated
